.github/workflows/build-windows.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         id: download-sdk
         uses: suisei-cn/actions-download-file@v1
         with:
-          url: https://github.com/freeorion/freeorion-sdk/releases/download/v12/FreeOrionSDK_12_MSVC-v141-Win32.zip
+          url: https://github.com/o01eg/freeorion-sdk/releases/download/v13/FreeOrionSDK_13_MSVC-v141-Win32.zip
```

Comment:
No reason not to use the v142 toolset SDK, I think...?

Reply:
Travis CI builds SDK with v141 toolset.

Reply:
It can be changed, I think? https://github.com/freeorion/freeorion-sdk/blob/master/.travis.yml#L75

Reply:
Travis CI uses older Windows with `Visual Studio 15`.